TRIM_E: 0
TRIM_A: 0
TRIM_R: 0
TRIM_T: 0.20

COURSE_KP: 1.35
COURSE_KD: -0.023
COURSE_KI: 0.0

ROLL_KP: 0.25
ROLL_KD: -0.07
ROLL_KI: 0

PITCH_KP: 0.30
PITCH_KD: -0.043
PITCH_KI: 0.0002
PITCH_FF: 0.0

AS_PITCH_KP: 2.0
AS_PITCH_KD: -0.2
AS_PITCH_KI: 0


AS_THR_KP: 3.2
AS_THR_KD: 0
AS_THR_KI: 0.0


ALT_KP: 0.12
ALT_KD: -0.00045
ALT_KI: 0.0000005


BETA_KP: -0.1164
BETA_KD: 0
BETA_KI: -0.0037111


CHI_INFTY: 1.5
K_PATH: 0.027
K_ORBIT: 1.2

CLIMB_THROTTLE: 0.6
CLIMB_ANGLE_DEG: 15.0
